Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -348,6 +358,16 @@ open class NavigationMapView: UIView {
         mapView.mapboxMap.onNext(.styleLoaded) { [weak self] _ in
```

Comment:
Can the style be loaded twice during the lifetime of the NavigationMapView?

Reply:
In this case no. As per Mapbox Maps docs for this method:
```
This will observe the next (and only the next) event of the specified type. After observation, the underlying subscriber will unsubscribe from the map or snapshotter.
```

This happens in `NavigationMapView.setupMapView(_:navigationCameraType:tileStoreLocation:)` method, which is called once.